### ISSUE TITLE
Disappears from Extension Menu

### ISSUE THREAD

**Issue by jkears** [microsoft/vscode-tye#201]
Type: <b>Bug</b>

Just recently this started, where when I restart VS Code, my Tye extension no longer shows in the side Extension Menu.  To restore to menu, I have to Disable the Enable Tye extension. 

Extension version: 0.5.1
VS Code version: Code 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:32:34.589Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 5950X 16-Core Processor             (32 x 3394)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|127.92GB (103.33GB free)|
|Process Argv|--folder-uri file:///c%3A/NextWare/CoreServices/MetaModelServices --crash-reporter-id d2f16950-6b88-43bc-80a3-5c78e3241d18|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627cf:30244335
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848cf:30577861
nodejswelcome1cf:30587006
3biah626:30602489
89544117:30613380
pythonsymbol12:30657548
a9j8j154:30646983
vsctsb:30662444

```

</details>

**Comment by karolz-ms** [comment]
@jkears are you still experiencing this issue? In other words, do you have to disable, then enable Tye extension regularly to make it work for you?

**Comment by jkears** [comment]
@karolz-ms , yes I still need to disable then re-enable.  This is required everyone I restart VSCode

**Comment by karolz-ms** [comment]
@jkears thank you! I am not going to pretend that I have an idea what is going on here because I don't 😕 

I took a look at this extension manifest and everything seems to be OK.

Two things worth checking:
1. Anything suspicious in the logs? to `F1 > Open View... > Shared` and `F1 > Open View... > Window` and check if you see anything related to Tye extension that seems out of order. If not, you can enable trace logging (`F1 > Set Log Level... > Trace`), restart VS Code and check the same thing again just to be sure.
2. Open extension folder (`F1 > Open Extensions Folder`) and open `extensions.json` file there. Double check that the path and other settings related to Tye extension seem OK.

Hope this gives us some clues!
